src/tests/libecalc/dto/test_generator_set.py

```diff
@@ -66,6 +66,9 @@ def test_valid(self):
             )
         }
 
+    @pytest.mark.skip(
+        reason="Due to circular dependencies we cannot specify a pump as an alternative as a consumer for genset, therefore we must set Any, which makes this not fail."
```

Comment:
How do we fix it?

Reply:
every case must be considered separately. Here we must figure out the problem with the pumpcharts I think. We can however not move those to domain yet, as that will cause serialization to not work (unless we add that, but it doesnt make sense to serialize domain objects...). So we must avoid need for serialization of the ecalcmodel first...I think.